src/middlewared/middlewared/plugins/pool.py

```diff
@@ -1372,7 +1375,7 @@ async def import_pool(self, job, data):
             for name, ids in (await self.middleware.call('keyvalue.get', key)).items():
                 for delegate in PoolDatasetService.attachment_delegates:
                     if delegate.name == name:
-                        attachments = await delegate.query(pool['path'], False)
+                        attachments = await delegate.query(pool['path'], False, {'locked': False})
```

Comment:
Why `{'locked': False}`?

Reply:
Motivation was to not try enabling attachments which are still locked but I think we should be at least toggling them which is what the method currently does. I'll make amends :)